domains/certificates/Query.API/API/DataSyncSyncer/DataSyncSyncerWorker.cs

```diff
@@ -150,3 +129,29 @@ private async Task SleepToNearestHour(CancellationToken cancellationToken)
             )
             .ToList();
 }
+
+public static class SynchronizationMigration
+{
+    public static async Task<int> MigrateSynchronizationPosition(this IDocumentStore store, CancellationToken cancellationToken)
```

Comment:
Is this something temporary just to clean up the db?

Reply:
Yes. It will be deleted on the next release.

Reply:
And so will the corresponding test